pages/products/[id].vue

```diff
@@ -49,7 +49,7 @@ onMounted(async () => {
     }
 });
 function handleUpdate() {
-    router.push({ path: `/updateProduct/${productId}` })
+    router.push({ path: `/products/update-product/${productId}` })
```

Comment:
why `/products/update-product` ? it contains product twice.
Shouldn’t it be
`/products/update/$productId` or
`/products/$productId/update`

Reply:
Resolved